### PR TITLE
Updating some test deps

### PR DESCRIPTION
**Pull request by jglick**
Supersedes #57. Picks up https://github.com/jenkinsci/git-client-plugin/pull/381.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -69,8 +69,8 @@
     <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.107.3</jenkins.version>
     <java.level>8</java.level>
-    <scm-api.version>2.2.7</scm-api.version>
-    <git-plugin.version>4.0.0-beta3</git-plugin.version>
+    <scm-api.version>2.3.2</scm-api.version>
```
Comment (stephenc) [inline pom.xml]: 2.4.0 probably now

Comment (jglick) [reply pom.xml]: As you prefer.

File: pom.xml [hunk 2]
```diff
@@ -69,8 +69,8 @@
     <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.107.3</jenkins.version>
     <java.level>8</java.level>
-    <scm-api.version>2.3.2</scm-api.version>
-    <git-plugin.version>3.3.0</git-plugin.version>
+    <scm-api.version>2.4.0</scm-api.version>
+    <git-plugin.version>4.0.0-rc</git-plugin.version>
```
Comment (rsandell) [inline pom.xml]: Not sure that I'm very comfortable with depending on an "rc" version of a plugin that is also removed from the update center?

Comment (jglick) [reply pom.xml]: This is for `test` scope.

**Pull request merged**